packages/storage-ui/src/Components/Modules/FilesList/FilesList.tsx

```diff
@@ -429,6 +425,51 @@ const FilesList = () => {
     [selectedCids]
   )
 
+  // select item with SHIFT pressed
+  const handleSelectItemWithShift = useCallback(
+    (item: ISelectedFile) => {
+      // item already selected
+      const isItemAlreadySelected = selectedItems.find((s) => s.cid === item.cid && s.name === item.name)
```

Comment:
Given the name, I'd consider making this a boolean
```suggestion
      const isItemAlreadySelected = selectedItems
        .find((s) => s.cid === item.cid && s.name === item.name)
        .length > 0
```